README.md

```diff
@@ -156,6 +156,15 @@ sheet.to_xml
 sheet.to_yaml
 ```
 
+You can specified file as default argument to `#to_csv`: 
+
+    # puts csv format content into a specified file
+    sheet.to_csv(File.new("/dev/null"))
+
+also you can specify the custom separator for the content:
+
+    sheet.to_csv(separator: ":") # default separator is comma: ","      
+
```

Comment:
This documentation is inconsistent with rest of readme.

Reply:
yeah, thanks. Plugin in IDE idea showed it as same as ```ruby. Now it's fixed.